resources/analysisTools/bam2fastq/coreutilsSortFastqSingle.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
Is there a common code base for this and the sortpair file?

Reply:
They are structurally similar but not identical. I don't think there deduplication possible with a  reasonable amount of effort and without loss of clarity.